src/main/java/seedu/address/logic/commands/AddModuleCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_CODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_TITLE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.module.Module;
+
+/**
+ * Adds a module to Plannit.
+ */
+public class AddModuleCommand extends Command {
+
+    public static final String COMMAND_WORD = "add-module";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a module to Plannit."
+            + "Parameters: "
+            + PREFIX_MODULE_CODE + "MODULE_CODE "
+            + "[" + PREFIX_MODULE_TITLE + "MODULE_TITLE]\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_MODULE_CODE + "CS1231S "
+            + PREFIX_MODULE_TITLE + "Discrete Structures";
+
+    public static final String MESSAGE_SUCCESS = "New module added: %1$s";
```

Comment:
```suggestion
    public static final String MESSAGE_ADD_MODULE_SUCCESS = "New module added: %1$s";
```
I noticed that the success message for deleting modules was named `MESSAGE_DELETE_MODULE_SUCCESS`. Should this be correspondingly named `MESSAGE_ADD_MODULE_SUCCESS`?

Reply:
Thanks so much @teoyuqi for spotting this code quality issue!